blueocean-core-js/src/js/components/RunButton.jsx

```diff
@@ -62,43 +46,9 @@ export class RunButton extends Component {
         }
     }
 
-    _onJobEvent(runData, event) {
-        const name = decodeURIComponent(
-            event.job_ismultibranch ? event.blueocean_job_branch_name : event.blueocean_job_pipeline_name
-        );
-        const runId = event.jenkins_object_id;
-
-        if (event.jenkins_event === 'job_run_started') {
-            const runDetailsUrl = buildRunDetailsUrl(runData);
-
-            toastService.newToast({
-                text: `Started "${name}" #${runId}`,
-                action: 'Open',
-                onActionClick: () => {
-                    if (this.props.onNavigation) {
-                        this.props.onNavigation(runDetailsUrl);
-                    }
-                },
-            });
-        } else if (event.jenkins_event === 'job_run_ended' && runData.result === 'ABORTED') {
-            toastService.newToast({
-                text: `Stopped "${name}" #${runId}`,
-            });
-        }
-    }
-
-    _filterJob(event) {
-        return event.blueocean_job_rest_url === this.props.runnable._links.self.href;
-    }
-
     _onRunClick() {
-        runApi.startRun(this.props.runnable);
-
-        const name = this.props.runnable.name;
-
-        toastService.newToast({
-            text: `Queued "${name}"`,
-        });
+        runApi.startRun(this.props.runnable)
```

Comment:
Probably the most significant change above: since we create "Started" and "Stopping" toasts immediately when the REST API call returns, we don't need to listen to SSE to create these toasts. The old code was a lot more complex than it needed to be.